root-packages/runc/build.sh

```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://www.opencontainers.org/
 TERMUX_PKG_DESCRIPTION="A tool for spawning and running containers according to the OCI specification"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.2.5"
-TERMUX_PKG_REVISION=1
-TERMUX_PKG_SRCURL=https://github.com/opencontainers/runc/archive/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=1ae11980477b789732915dfd7ba4246f5babe5ff76a31021d36365007d9713b0
+TERMUX_PKG_VERSION="1.3.0~rc.1"
+_REAL_VERSION="${TERMUX_PKG_VERSION/\~/-}"
```

Comment:
Emmm... It is not enough just change it here. `termux_pkg_auto_update` should be overriden too.

https://github.com/termux/termux-packages/blob/14fb540d74d13df2379b4aa9b725e21f258358a1/x11-packages/wine-devel/build.sh#L86-L107

Reply:
not applicable anymore